src/apis/member/query.ts

```diff
@@ -0,0 +1,10 @@
+import { useQuery } from '@tanstack/react-query'
+import { getCookie } from 'cookies-next'
+import { getMemberProfile } from './api'
+
+export const useGetMemberProfileQuery = () =>
+  useQuery({
+    queryKey: ['member'],
+    queryFn: getMemberProfile,
+    enabled: Boolean(getCookie('accessToken'))
+  })
```

Comment:
```suggestion
export const useGetMemberProfileQuery = (accessToken) =>
  useQuery({
    queryKey: ['member'],
    queryFn: getMemberProfile,
    enabled: Boolean(accessToken)
  })
```
getCookie를 query 내부에서 의존하고 있어서  디버깅이 어려워질 것 같아 외부에서 토큰을 주입받아서 사용하는 형태가 더 좋을 것 같습니다!

Reply:
홍 그렇겠네요 ! 수정하겠습니다 !

Reply:
[토큰 param으로 받기](https://github.com/price-offer/offer-fe/pull/175/commits/e1b19a97ac215f4be7d4b0ecd8e7b06d0c7aa3ed) + [queryKey 변경](https://github.com/price-offer/offer-fe/pull/175/commits/b0611408b4c0776cda6c483d5e8dc7ac798e0401) 완료!